doc/examples/realtime_servo/realtime_servo_tutorial.rst

```diff
@@ -51,6 +51,23 @@ Expected Output
 
 Note that the controller overlay here is just for demonstration purposes and is not actually included
 
+Introspection
+-------------
+
+Here are some tips for inspecting and/or debugging the system.
+
+View the ros2_controllers that are currently active with :code:`ros2 control list_controllers`. You will see a JointTrajectoryController that receives the joint position commands from Servo and handles them in the simulated robot driver. Servo is also compatible with JointGroupPosition or JointGroupVelocity-type controllers.
+
+:code:`ros2 topic echo /servo_node/status` shows the current state of the Servo node. If :code:`0` is published, all is well. The definition for all enums can be seen :moveit_codedir:`here.<moveit_ros/moveit_servo/include/moveit_servo/status_codes.h>`
+
+:code:`ros2 node list` shows the following. :code:`ros2 node info` can be used to get more information about any of these nodes.
+
+- :code:`joy_node` handles commands from the XBox controller
```

Comment:
Maybe make all of the tips bullet points or none of them bullet points

Reply:
Thanks @mlautman, hope life is good

Reply:
and shame on you for not having a Linux machine